internal/command.go

```diff
@@ -139,9 +140,6 @@ func NewConfluentCommand(cfg *config.Config) *cobra.Command {
 	if cfg.IsTest || featureflags.Manager.BoolVariation("cli.flink", cfg.Context(), config.CliLaunchDarklyClient, true, false) {
 		cmd.AddCommand(flink.New(cfg, prerunner))
 	}
-	if cfg.IsTest || featureflags.Manager.BoolVariation("custom-connect.cli.custom_code_logging.early_access", cfg.Context(), config.CliLaunchDarklyClient, true, false) {
```

Comment:
custom-connect.x flag should not working with cli. remove this flag from CLI

Reply:
we have backend flag to control this feature. and this feature is OK to expose to all customers

Reply:
@byang-confluent Do we also want to expose the command reference documentation with this change?

Reply:
@sgagniere , the public doc is not ready yet, we have ticket to track https://confluentinc.atlassian.net/browse/DOCS-31145